schema/schema.go

```diff
@@ -50,6 +51,16 @@ type Column struct {
 	Ignored Ignored
 }
 
+// Fkey respresents a foreign key.
+type Fkey struct {
```

Comment:
Use a full name `ForeignKey`?

Reply:
`PrimaryKeys []Key` - This is how primary keys are defined in schema.go file. To follow similar structure, I have defined it as `ForeignKeys []Fkey`. Also, `schema.Fkey` is accessed in multiple file - so it would be a good idea to keep it short.

Let me know what are your views.

Reply:
I think ForeignKey would be better.
In general, names of things in public interfaces are not abbreviated too aggressively.
In contrast, we aggressively shorten names of local variables, because their context is very local. But public interfaces have to make sense from afar, so are generally more self descriptive.

We do abbreviate Column to Col (ColNames, ColDefs) in schema.go -- I think this is ok because for databases this is a fairly standard abbreviation. But I think abbreviating Foreign to F is not standard and is too confusing.

Reply:
Ok. I have updated it to `ForeignKey`.